pre-commit.sh

```diff
@@ -9,34 +9,44 @@ function header
     echo
 }
 
+SOURCE_FILES=b2/b2.py
+
 header Checking Formatting
 
-if yapf b2 > b2.yapf.out
-then
-    echo "Formatting is good."
-    rm b2.yapf.out
-else
-    echo "Formatting updated:"
-    echo
-    diff b2 b2.yapf.out
-    mv b2.yapf.out b2
-    chmod +x b2
-fi
+for src_file in $SOURCE_FILES
+do
+    echo $src_file
```

Comment:
Please **always** quote strings in bash. Otherwise random stuff stops working when a space appears in a variable or a file name.